### PR TITLE
Fixed exception message

### PR DESCRIPTION
**Pull request by islem-esi**
#### Brief description of what is fixed or changed
Changed exception messages in two files (one is wrong an the other is vague)

NO ENTRY

### CODE REVIEW

**Review by dyc3 — APPROVED** [review]

**Comment by oscargus** [comment]
Looks good!

See https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file
to get rid of the failing test (add your name and email to the mailmap file).

File: .mailmap [hunk 1]
```diff
@@ -1325,3 +1325,4 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Øyvind Jensen <jensen.oyvind@gmail.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
+Islem BOUZENIA <fi_bouzenia@esi.dz>
```
Comment (oscargus) [inline .mailmap:1328]:
```suggestion
Islem BOUZENIA <fi_bouzenia@esi.dz> Islem BOUZENIA <61057090+islem-esi@users.noreply.github.com>
```

Comment (oscargus) [reply .mailmap]: Seems like you had a previous entry if I interpreted the error message correctly. I do not really follow this, but you should have a line similar to the above, with both your email addresses.

Comment (oscarbenjamin) [reply .mailmap]: The message is confusing but the problem is that neither of the names here actually matches what is in the git commits (`islem-esi <fi_bouzenia@esi.dz>`). The line to be added should be:
```
Islem BOUZENIA <fi_bouzenia@esi.dz> islem-esi <fi_bouzenia@esi.dz>
```
There the name "islem-esi" is what is actually in the commits.

After adding that line the `bin/mailmap_check.py` script should be run to sort the file.

**Pull request merged**